packages/packagers/ts/src/TSPackager.js

```diff
@@ -13,11 +13,22 @@ export default new Packager({
     assert.equal(assets.length, 1, 'TS bundles must only contain one asset');
     let code = await assets[0].getCode();
     let map = await assets[0].getMap();
-    if (map) {
-      let sourceMapReference = await getSourceMapReference(map);
-      code += '\n//# sourceMappingURL=' + sourceMapReference + '\n';
-    }
 
     return {contents: code, map};
   },
+  async postProcess({contents, map, getSourceMapReference}) {
+    // $FlowFixMe sketchy null checks are fun
+    if (!map) return {contents, map};
```

Comment:
do you mean `if (map == null || map === '')`?

Reply:
I guess I mean `map === null || map === undefined`

Reply:
> map === null || map === undefined

that would be `map != null`

Reply:
I'll update it...

Reply:
Just tried it out and now I remember why I did it like I did, flow says it can still be null or undefined when using `map != null` it might be because of the $FlowFixMe once I remove that it complains. So I'll just leave it like that for now...